### PR TITLE
50 animate readme

### PR DESCRIPTION
**Pull request by acse-ej321**
Added a README.md with:
- a brief package description pulled from Pyroteus and Firedrake documentation
- installation instructions: moderately verbose as a first attempt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# animate
+Anisotropic mesh adaptation toolkit for Firedrake
+
+
+    The implementation of metric-based mesh adaptation used in PETSc assumes that the
+    metric is piece-wise linear and continuous, with its degrees of freedom at the
+    mesh vertices.
```
Comment (jwallwork23) [inline README.md:7]: Perhaps put this paragraph after the next one since it is more about details? Also, don't use code styling.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Animate is an mesh adaptation toolkit enabling users to anisotropically adapt a mesh based on a Riemannian metric framework with control over the shape, orientation and size of the resulting mesh elements. Animate is built for use with Firedrake and  strongly leverages PETSc DMPlex functionality. The implementation of metric-based mesh adaptation used in PETSc assumes that the metric is piece-wise linear and continuous, with its degrees of freedom at the mesh vertices.
+
+For more information on Firedrake, please see: [Firedrake Website](https://www.firedrakeproject.org/).
+For more information on PETSc, please see: [PETSc metric based mesh adaptation](https://petsc.org/release/docs/manual/dmplex/#metric-based-mesh-adaptation)
+
+## Linux Installation
+
+The following installation instructions assume a linux or WSL operating system. The options below include installation from a custom shell script which also installs the custom setup for Firedrake and PETSc. If Firedrake is already installed, please see instructions to install Animate via git clone.
```
Comment (jwallwork23) [inline README.md:16]: Capital L for Linux.

Comment (jwallwork23) [inline README.md:16]: `git clone`, i.e. use code styling.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### To install Animate with all in one shell script
+
+The 'all-in-one' shell script will install Animate and all its key dependencies, including Firedrake and PETSc.
+
+Instructions:
+- Download installation files either:
+	-  manually from: or via curl -O `<filename>`
```
Comment (jwallwork23) [inline README.md:24]: `curl -O <filename>`, i.e., code styling

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		- install/install_firedrake_custom_mpi.sh
+		- /install/petsc_options.txt
```
Comment (jwallwork23) [inline README.md:26]: Code styling and remove initial `/`.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+	- via curl:
+		- curl -O https://raw.githubusercontent.com/pyroteus/animate/50_animate_readme/install/install_firedrake_custom_mpi.sh
+		- curl -O https://raw.githubusercontent.com/pyroteus/animate/50_animate_readme/install/petsc_options.txt
```
Comment (jwallwork23) [inline README.md:29]: Code styling. Also put `main` instead of this branch name.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Install animate via `make animate`
+- Execute the test suite to confirm installation was successful via `make test`
+
+### To install Animate via Docker image
+
+The instructions for installing the Animate Docker image can be found in the `Docker` folder, including a makefile for installation and execution.
+
+Alternatively, manually install the Animate docker image:
+- Pull the docker image: `docker pull jwallwork/firedrake-parmmg`
+- Run the docker image: `docker run --rm -it -v ${HOME}:${HOME} jwallwork/firedrake-parmmg`
```
Comment (jwallwork23) [inline README.md:40]: Perhaps add a note beneath that by using this command you are giving Docker access to your home space.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+
+### To install Animate via git clone
+Installing Animate via cloning the GitHub repository assumes prior installation of Firedrake and its dependencies. For separate instructions for installing Firedrake please see: [Firedrake download instructions](https://www.firedrakeproject.org/download.html)
```
Comment (jwallwork23) [inline README.md:44]: Full stop.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+To install Animate via git clone:
+- from the main Animate GitHub, Clone the repository using HTTPS or SSH
```
Comment (jwallwork23) [inline README.md:47]: No capital C for clone. Also full stop.

File: README.md (hunk 1 above)
Comment (acse-ej321) [reply README.md:7]: I borrowed the description from the previous meshadapt.py implementation in Firedrake for continuity. You are right, I incorporated it into the overall description and forgot to delete the original copy.

File: README.md (hunk 2 above)
Comment (acse-ej321) [reply README.md:16]: I've done this in both the title and the body of the text

File: README.md (hunk 4 above)
Comment (acse-ej321) [reply README.md:24]: done

**Review by jwallwork23 — APPROVED** [review]

**Pull request merged**